modules/33acrossAnalyticsAdapter.js

```diff
@@ -300,6 +300,22 @@ function createReportFromCache(analyticsCache, completedAuctionId) {
     report.usPrivacy = uspDataHandler.getConsentData();
   }
 
+  const gdprConsentData = gdprDataHandler.getConsentData();
+  const gdprApplies = Boolean(gdprConsentData?.gdprApplies);
+  report.gdpr = Number(gdprApplies);
+  if (gdprApplies) {
```

Comment:
I think in a few slack conversations it has been mentioned that the gdprApplies flag isn't very reliable, and that we should send the GDPR consent string regardless of the value of the flag. So the `If` statement is not needed. Keep the two parameters but don't have the condition.
```
report.gdpr = Number(gdprApplies);
report.gdprConsent = gdprConsentData.consentString || '';
```

There's actually another PR, that hasn't been merged yet, about removing the condition in the 33x UIM

Reply:
fixed